pkg/informer/informer_test.go

```diff
@@ -103,11 +104,6 @@ func TestBuiltInInformableTypes(t *testing.T) {
 	}
 
 	allKnownTypes := kcpscheme.Scheme.AllKnownTypes()
-
-	// CRDs are not included in the genericcontrolplane scheme (because they're part of the apiextensions apiserver),
-	// so we have to manually add them
-	allKnownTypes[schema.GroupVersionKind{Group: "apiextensions.k8s.io", Version: "v1", Kind: "CustomResourceDefinition"}] = reflect.TypeOf(struct{}{})
```

Comment:
isn't that still the case?

Reply:
I don't think so. We add this to schema bellow? No?